worlds/banjo_tooie/Options.py

```diff
@@ -0,0 +1,513 @@
+
+class EnableMultiWorldDinoRoar(Toggle):
+    """Baby T-Rex Roar's is lost across the MultiWorld. Other players need to help him learn to ROAR!"""
+    display_name = "Baby T-Rex Roar"
+
+class EnableNestsanity(Toggle):
+    """Eggs and feather nests give checks when you collect them for the first time. They behave as regular egg nests after they have been collected."""
+    display_name = "Nestsanity"
+    
+class TrapsToNestRatio(Range):
+    """Select a percentage of feather and egg nests items to be replaced with trap items
+    Requires Traps and Nestsanity to have an effect."""
+    display_name = "Traps to Nests Ratio"
+    range_start = 0
+    range_end = 100
+    default = 0
+    
+class GoldenEggsWeight(Range):
+    """The weight of Golden Eggs in the trap pool.
+    Requires Traps and Nestsanity to have an effect"""
+    display_name = "Golden Eggs Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+
+class TripTrapWeight(Range):
+    """The weight of Trip Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Trip Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+    
+class SlipTrapWeight(Range):
+    """The weight of Slip Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Slip Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+    
+class TransformTrapWeight(Range):
+    """The weight of Transform Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Transform Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 40
+    
+class SquishTrapWeight(Range):
+    """The weight of Squish Traps in the trap pool.
+    Requires Traps to have an effect"""
+    display_name = "Squish Trap Weight"
+    range_start = 0
+    range_end = 100
+    default = 20
+    
+class KingJingalingHasJiggy(DefaultOnToggle):
+    """King Jingaling will always have a Jiggy for you."""
+    display_name = "King Jingaling Jiggy"
+
+class SkipPuzzles(DefaultOnToggle):
+    """Open world entrances without having to go to Jiggywiggy."""
+    display_name = "Skip Puzzles"
+
+class Backdoors(Toggle):
+    """Opens many one-way switches on game start, allowing for more backdoor access to levels.
+    The following gates are preopened: MT -> TDL, MT -> HFP, GGM -> WW, WW -> TDL.
+    For MT -> TDL, only the gate accessed from TDL's side is opened. For GGM -> WW, the boulders are still intact.
+    The bridge from HFP's entrance is pre-moved to allow secondary access to Cliff Top.
+    George is pre-dropped to make HFP -> JRL more accessible."""
+    display_name = "Open Backdoors"
+
+class OpenHag1(DefaultOnToggle):
+    """HAG 1 boss fight is opened when Cauldron Keep is opened, requiring fewer Jiggies to win."""
+    display_name = "HAG 1 Open"
+
+class RandomizeWorlds(Toggle):
+    """Worlds will open in a randomized order. Randomized Moves and Skip Puzzle Required."""
```

Comment:
```suggestion
    """Worlds will open in a randomized order. Randomized Moves and Skip Puzzles Required."""
```
The display name for the other option has an s at the end

Reply:
yep